### DISCUSSION TITLE
"Permission denied" Error Message when running brew services start/restart httpd

### DISCUSSION THREAD

**Discussion by laurencereeds** [Homebrew/discussions#538] [Getting started] [answered]
Hi,

I want to set up an Apache server. I had to previously debug my HomeBrew after I made the Big Sur update.
I followed all the steps described on this [macOS Big Sur 11.0 Setup: Homebrew + Apache + PHP + MariaDB](https://gist.github.com/karlhillx/791babda83495eab6fbf64f8a3270f86) article from @karlhillx.
I made all the changes on the httpd.conf file with VSC and when I try to restart Apache I return the following error message:

"Error: Permission denied @ rb_sysopen - /Users/laurencereeds/Library/LaunchAgents/homebrew.mxcl.httpd.plist"

I have tried :

sudo chmod -R g+w /Library/Caches/Homebrew
-> chmod: /Library/Caches/Homebrew: No such file or directory

sudo chown -R ${USER} /Library/Caches/Homebrew/
-> "chown: /Library/Caches/Homebrew/: No such file or directory"

sudo chown -R $(whoami) /usr/local/*
-> I have still a permission denied

Can anyone please shed some light on what is happening? And what I should do?

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
You seem to have tried to run `sudo brew services start httpd`, you'll need to run `sudo brew services stop httpd` and then the start command again to restart apache.

**Reply by laurencereeds** [reply]
Thank you. It worked!
